docs/source/ingest/destination_connectors/pinecone.rst

```diff
@@ -0,0 +1,80 @@
+Pinecone
+===========
+
+Batch process all your records using ``unstructured-ingest`` to store structured outputs and embeddings locally on your filesystem and upload those to a Pinecone index.
+
+First you'll need to install the Pinecone dependencies as shown here.
+
+.. code:: shell
+
+  pip install "unstructured[pinecone]"
+
+Run Locally
+-----------
+The upstream connector can be any of the ones supported, but for convenience here, showing a sample command using the
+upstream s3 connector. This will create new files on your local.
+
+.. tabs::
+
+   .. tab:: Shell
+
+      .. code:: shell
+
+        unstructured-ingest \
+            s3 \
```

Comment:
Can we update this to use the same approach we already take using the local connector for the first half of this.

Reply:
Example: [mongodb.rst](https://github.com/Unstructured-IO/unstructured/blob/roman/mongo-db-atlas-dest-conn/docs/source/ingest/destination_connectors/mongodb.rst?plain=1#L24)

Reply:
Addressed with: [update docs](https://github.com/Unstructured-IO/unstructured/pull/1774/commits/c3266f0bc7a121c7b4bb813c86594656da4efe4a)